source/components/miniRecipe.js

```diff
@@ -79,12 +82,14 @@ class SearchRecipe extends HTMLElement {
         const timeNumb = document.createElement('p')
         timeNumb.setAttribute('class', 'recipe-time');
 
-        timeNumb.innerHTML = "14";
+        timeNumb.innerHTML = "<" + data.readyInMinutes + "min";
+        console.log(data.readyInMinute);
 
+        recipeImg.setAttribute('src',data.image);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8430255)
```suggestion
        recipeImg.setAttribute("src",data.image);
```